packages/style-kit/stories/typography.stories.js

```diff
@@ -1,42 +1,30 @@
 export default { title: 'Typography' };
 
-import 'style-loader!./typography.scss';
-
-const splitLabel = label => label.split('-').join(' ');
+import 'style-loader!./typography.stories.scss';
 
 export const font = () => `
 <p><strong>Euclid Circular B</strong>, by the Swiss Typefaces foundry is the official
 Meeco font to be used across all UI and marketing. To stay consistant, we only use
 the Regular, Medium and Bold weights.</p>
 `;
 
-export const headings = () => `
+export const typographicStyles = () => `
 <h1>Heading 1</h1>
 <h2>Heading 2</h2>
 <h3>Heading 3</h3>
 <h4>Heading 4</h4>
 <h5>Heading 5</h5>
 <h6>Heading 6</h6>
-`;
 
-export const body = () => {
-  const container = document.createElement('div');
-
-  const fonts = ['large', 'medium', 'small'];
-  fonts.forEach(font => {
-    const text = document.createElement('p');
-    text.className = `${font}`;
-    text.innerHTML = `This is body text ${font}`;
-    container.appendChild(text);
-  });
-
-  const labels = ['card-label', 'input-label', 'input-value'];
-  labels.forEach(label => {
-    const text = document.createElement('p');
-    text.className = `${label}`;
-    text.innerHTML = splitLabel(label);
-    container.appendChild(text);
-  });
-
-  return container;
-};
+<p>Normal Body Text (Paragraph)</p>
+
+<p class="large">Body Text Large</p>
+<p class="medium">Body Text Medium</p>
+<p class="small">Body Text Small</p>
+
+<p class="card-label">Card Name Label</p>
+
+<label>Text Field Label</label>
+
+<input type="text" value="Text Field Value" />
+`;
```

Comment:
@izzypesk BTW I pulled the loops out of here just to make it a bit easier to see how each font is used from a code perspective (since this is sort of the documentation of how each font style is added in the front end).